### PR TITLE
Pending loan requests

### PR DESCRIPTION
**Pull request by Bishop19**
This PR:
- adds admin dashboard to manage pending loan requests (#11)

We need to protect this admin routes somehow (maybe another domain?)

### CODE REVIEW

File: web/src/components/Header.js [hunk 1]
```diff
@@ -8,9 +8,29 @@ const Header = (props) => {
   const { auth, onLogout } = props;
 
   const nav_items = auth ? (
-    <Grid item>
-      <NavMenu onLogout={onLogout} />
-    </Grid>
+    <>
```
Comment (pferreira101) [inline web/src/components/Header.js]: Why do you need a fragment here?

File: web/src/containers/LoanRequest.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  withStyles,
+  InputAdornment,
+} from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import LoanRequestService from "../services/loanrequest.service";
+import { useHistory } from "react-router";
+
+const styles = {
```
Comment (pferreira101) [inline web/src/containers/LoanRequest.js:21]: We are once again repeating this block of CSS. Shouldn't this be moved to the layout?

**Comment by pferreira101** [comment]
This seems to be working as intended.  We need to work on the navbar because on smaller screens the items are displayed in an oddly way. Regarding the Admin page, we could implement role-based auth.  Here is as [example](https://jasonwatmore.com/post/2019/02/01/react-role-based-authorization-tutorial-with-example). I think this would suffice.

**Review by pferreira101 — APPROVED** [review]

File: web/src/components/Header.js (hunk 1 above)
Comment (Bishop19) [reply web/src/components/Header.js]: in ternary expressions you can only return one element, so i've wrapped the two Grids

**Review by Tiagod25 — APPROVED** [review]

**Pull request merged**